tracing-subscriber/src/fmt/fmt_subscriber.rs

```diff
@@ -490,6 +490,36 @@ where
         }
     }
 
+    /// Sets whether or not the binary name of the process is displayed when
+    /// formatting events. If binary_name is None, argv\[0\] will be used,
+    /// otherwise the spcified string will be used.
+    ///
+    /// [argv\[0\]]: std::env::args
+    pub fn with_binary_name(
+        self,
+        display_binary_name: bool,
+        binary_name: Option<&str>,
```

Comment:
To make the api more inclusive (and maybe avoid an additional allocation), we could use `Option<impl Into<String>>` here.

Reply:
Done.